.github/workflows/deploy-auth-api.yml

```diff
@@ -6,17 +6,22 @@ env:
   AUTH_API_CLUSTER: shared-cluster
   AUTH_API_SERVICE: auth-api
 
+# Required for IDP JWT and actions/checkout
+permissions:
+      id-token: write
+      contents: read
+   
 jobs:
   deploy:
     name: Deploy latest stable image
     environment: shared
     runs-on: ubuntu-latest
     steps:
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v4
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1.7.0
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
```

Comment:
Unclear why this diff is showing as it's already in main, but it's the correct change anyway